Description: Drop -Werror from AM_INIT_AUTOMAKE and handle parallel tests
 This avoids problems with the version of Gnulib used here causing warnings
 from Automake 1.13.  This patch can be dropped at the next upstream
 release.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: not-needed
Last-Update: 2013-06-03

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -4,7 +4,7 @@
 # Initialise and check we're in the correct directory.
 AC_INIT([man-db], [2.6.3], [cjwatson@debian.org])
 AC_CONFIG_AUX_DIR([tools])
-AM_INIT_AUTOMAKE([1.10 -Wall -Wno-override -Werror foreign dist-xz no-dist-gzip])
+AM_INIT_AUTOMAKE([1.11 -Wall -Wno-override foreign dist-xz no-dist-gzip parallel-tests])
 AM_MAINTAINER_MODE
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AC_PREREQ([2.59])
Index: b/man/Rules.man
===================================================================
--- a/man/Rules.man
+++ b/man/Rules.man
@@ -25,7 +25,7 @@
 
 export top_builddir LINGUA EGREP
 if !CROSS_COMPILING
-TESTS_ENVIRONMENT = @SHELL@ $(top_srcdir)/man/check-man
+LOG_COMPILER = @SHELL@ $(top_srcdir)/man/check-man
 TESTS = $(man1_MANS) $(man5_MANS) $(man8_MANS)
 endif
 
Index: b/src/tests/Makefile.am
===================================================================
--- a/src/tests/Makefile.am
+++ b/src/tests/Makefile.am
@@ -18,9 +18,12 @@
 ## along with man-db; if not, write to the Free Software Foundation,
 ## Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
-TESTS_ENVIRONMENT = PATH=..:$$PATH DBTYPE=$(DBTYPE) \
-		    top_builddir=$(top_builddir) \
-		    @LOCALCHARSET_TESTS_ENVIRONMENT@ $(SHELL)
+TESTS_ENVIRONMENT = PATH=..:$$PATH; export PATH; \
+		    DBTYPE=$(DBTYPE); export DBTYPE; \
+		    top_builddir=$(top_builddir); export top_builddir; \
+		    @LOCALCHARSET_TESTS_ENVIRONMENT@; export CHARSETALIASDIR;
+# Each test must use the configure-detected shell, not necessarily /bin/sh.
+AM_LOG_FLAGS = $(SHELL)
 ALL_TESTS = \
 	lexgrog-1 \
 	man-1 man-2 man-3 \
